common/conf/paths.hpp

```diff
@@ -24,6 +24,8 @@ namespace paths {
 
 using namespace std;
 
+string JoinPaths(const string &prefix, const string &suffix);
```

Comment:
S:
`paths::Join` should be plenty I think. In case we will probably use this other places as well.

Reply:
+1

Reply:
:+1: